api/docs/release.dox

```diff
@@ -145,7 +145,9 @@ compatibility changes:
 
 Further non-compatibility-affecting changes include:
 
- Nothing yet.
+ - Adds drmgr_register_opcode_instrumentation_event() and
```

Comment:
nit: All the lists below use the past tense here, so s/Adds/Added/ for consistency.

Reply:
Done.